src/fonduer/learning/disc_models/lstm.py

```diff
@@ -16,12 +16,14 @@
 
 class LSTM(NoiseAwareModel):
```

Comment:
Same comment here. How does user initialize this?
```
class fonduer.learning.LSTM(seed=123, **kwargs)
```
What are kwargs? Can we remove or enumerate?